DiffPlex/DiffBuilder/Model/DiffPaneModel.cs

```diff
@@ -5,6 +5,8 @@ namespace DiffPlex.DiffBuilder.Model
     public class DiffPaneModel
     {
         public List<DiffPiece> Lines { get; }
+        
+        public bool? HasDifferences { get; internal set; }
```

Comment:
Actually @michasacuer would it simplify everything if HasDifferences was a readonly property that actually computed
`.Lines.Any(x => x.Type != ChangeType.Unchanged)`

Reply:
Nice concept tbh, sorry it takes 3 commits to get it right ;p